core/src/components/item-divider/item-divider.scss

```diff
@@ -30,13 +30,24 @@
   --inner-padding-start: 0px;
 
   @include font-smoothing();
+
+  /* stylelint-disable */
+  @include ltr() {
+    padding-top: var(--padding-top);
```

Comment:
Padding top and bottom don't need to be directional. You should be able to move those above the `ltr`/`rtl` mixins like:

```scss
@include padding(var(--padding-top), null, var(--padding-bottom), null);
```
```